### ISSUE TITLE
It's more of a sugestion, please make batch video possible

### ISSUE THREAD

**Issue by taircn** [thygate/stable-diffusion-webui-depthmap-script#97]
It is amazing thing to try, people in [/r/CrossView/ ](https://www.reddit.com/r/CrossView/comments/10i66l7/there_is_an_option_in_depthmap_script_for_neural/) absolutely loved it and i have a sugestion, Right now there is an option in Depth-script UI that does 4 demo videos, but it doesn't work well for batch , or batch directory, as it doesn't take into consideration settings for video stream.
It would be great to have an option to autoproceed to make a 1 video file with settings made by user instead of 4 short demo ones. Or a choice. 
Thank you!

**Comment by thygate** [comment]
So instead of the 4 demo videos, just create one video with the settings in the 'create video' setting .. ?

**Comment by taircn** [comment]
> 



> So instead of the 4 demo videos, just create one video with the settings in the 'create video' setting .. ?

Yeah. That would be much more useful, especially for "batch from directory" mode. Length of video matters a lot, and demo doesn't take it into consideration. Overall it's fantastic addon, thank you for your work!

**Comment by kornerson** [comment]
this would be a fabulous addition. This extension needs more exposure. Im amazed with it's results

**Comment by Vendaciousness** [comment]
I wasted a lot of time in Deforum, trying to get consistent AI video output with minimal changes/flicker, but this script is so much more practical from a view for making AI generative video!  All we need are some small changes like this and we'll be set!
